ModeratorFrontEnd/AIForOrcas/AIForOrcas.Server/Controllers/DetectionsController.cs

```diff
@@ -236,23 +238,31 @@ public async Task<IActionResult> GetUnreviewedAsync([FromQuery] DetectionQueryPa
 				// total number of records
 				double recordCount = queryable.Count();
 
+				var results = queryable
+					.Select(x => DetectionProcessors.ToDetection(x)).ToList();
```

Comment:
Is ToList necessary here? (see relevant comments in DetectionProcessors.cs below)

Reply:
Yes. See conversation above.